lib/ASRouterPreferences.jsm

```diff
@@ -50,6 +56,17 @@ class _ASRouterPreferences {
     }, []);
   }
 
+  _migratePrefs() {
+    for (let [oldPref, newPref] of MIGRATE_PREFS) {
+      if (!Services.prefs.prefHasUserValue(oldPref)) {
+        continue;
+      }
+      // If the pref was user modified we assume it was set to false
+      const oldValue = Services.prefs.getBoolPref(oldPref, false);
+      Services.prefs.setBoolPref(newPref, oldValue);
```

Comment:
I think you also need to clear the user value of the old pref to prevent this from happening each time